pdns/nsec3dig.cc

```diff
@@ -163,8 +163,9 @@ try
   set<DNSName> namesseen;
   set<DNSName> namestocheck;
   nsec3set nsec3s;
+  nsec3types nsec3t;
   string nsec3salt;
-  int nsec3iters = 0;
+  int nsec3iters = -1;
```

Comment:
That sounds good but the implications of this change are not obvious to me :-)

Reply:
ah, I was trying to add detection of changed params but it was going to be ugly, I forgot to take this bit out. I'll fix that :)